server/src/instant/util/tracer.clj

```diff
@@ -166,21 +167,21 @@
            (end-span! *span*))))))
 
 ;; (XXX)
-;; Given a `sample-rate`, we will randomly skip spans at that rate 
+;; Given a `sample-rate`, we will randomly skip spans at that rate
```

Comment:
Nit. Should we delete this comment now?

Reply:
I think we should leave everything the same for a bit while we get comfortable with refinery, then come back and move any sampling configuration to the rules.yaml.

Reply:
Sounds good!